tests/functional/c/consider/consider_using_f_string.py

```
@@ -0,0 +1,107 @@
+    print("{Par_1}{Par_2}".format(Par_1=PARAM_1, Par_2=PARAM_2))  # [consider-using-f-string]
+    print("{Param_1}".format(*PARAM_LIST_SINGLE))  # [consider-using-f-string]
+    print("{Param_1}".format(**PARAM_DICT_SINGLE))  # [consider-using-f-string]
+    print("String %s" % (PARAM_1))  # [consider-using-f-string]
+    print("String %s %s" % (PARAM_1, PARAM_2))  # [consider-using-f-string]
+    print("String %s" % (PARAM_LIST_SINGLE))  # [consider-using-f-string]
+
+
+def statement_good():
+    "String {}, {} or {}".format(*PARAM_LIST)
+    "String {}, {}, {} or {}".format(*PARAM_LIST_SINGLE, *PARAM_LIST)
+    "String {Param}, {}, {} or {}".format(Param=PARAM_1, *PARAM_LIST)
```

Ruff (0.11.9)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)